packages/frint-store/src/createStore.spec.js

```diff
@@ -442,4 +443,80 @@ describe('frint-store › createStore', function () {
       subscription.unsubscribe();
     });
   });
+
+  it('creates Store with epics', function (done) {
+    // constants
+    const PING = 'PING';
+    const PONG = 'PONG';
+
+    const INITIAL_STATE = {
+      isPinging: false,
+    };
+
+    // reducers
+    function pingReducer(state = INITIAL_STATE, action) {
+      switch (action.type) {
+        case PING:
+          return {
+            isPinging: true,
+          };
+
+        case PONG:
+          return {
+            isPinging: false,
+          };
+
+        default:
+          return state;
+      }
+    }
+
+    const rootReducer = combineReducers({
+      ping: pingReducer,
+    });
+
+    // epics
+    function pingEpic$(action$) {
+      return action$
+        .filter(action => action.type === PING)
+        .delay(10)
+        .map(() => ({ type: PONG }));
+    }
+
+    const rootEpic$ = combineEpics(pingEpic$);
+
+    // Store
+    const Store = createStore({
+      enableLogger: false,
+      reducer: rootReducer,
+      epic: rootEpic$,
+    });
+
+    const store = new Store();
+
+    expect(store.getState().ping.isPinging).to.equal(false);
+
+    store.getState$()
+      .take(3)
+      .scan(
+        function (acc, curr) {
```

Comment:
unless we have single-line functions, or have functions that need access to its parent scope, we avoid using arrow functions everywhere in the Frint repo.